src/GlobalStyle.js

```diff
@@ -0,0 +1,72 @@
+import { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyle = createGlobalStyle`
+  @font-face {
+    font-family: 'NotoSansCJKKR';
+    font-weight: normal;
+    font-style: normal;
+    src: url(asset/fonts/NotoSansCJKkr/NotoSansCJKkr-Regular.otf) format('opentype');
+  }
+  @font-face {
+    font-family: 'NotoSansCJKKR';
+    font-weight: 500;
+    font-style: normal;
+    src: url(asset/fonts/NotoSansCJKkr/NotoSansCJKkr-Medium.otf) format('opentype');
+  }
+  @font-face {
+    font-family: 'NotoSansCJKKR';
+    font-weight: bold;
+    font-style: normal;
+    src: url(asset/fonts/NotoSansCJKkr/NotoSansCJKkr-Bold.otf) format('opentype');
+  }
+  
+  @font-face {
+    font-family: 'NotoSansKR';
+    font-weight: normal; 
+    font-style: normal;
+    src: url(asset/fonts/NotoSansKR/NotoSansKR-Regular.woff) format('woff');
+  }
+  @font-face {
+      font-family: 'NotoSansKR';
+      font-weight: 500;
+      font-style: normal;
+      src: url(asset/fonts/NotoSansKR/NotoSansKR-Medium.woff) format('woff');
+  }
+
+  @font-face {
+    font-family: 'NanumSquareRound';
+    font-weight: bold;
+    font-style: normal;
+    src: url(asset/fonts/other/NanumSquareRoundOTFB.otf) format('opentype');
+}
+
+
+  html, body, #root{
+    font-family: 'NotoSansCJKKR', 'Noto Sans KR', 'Noto Sans', sans-serif;
+    margin: 0;
+    height: 100%;
+    width: 100%;
+  }
+
+  /* reset */
+  body,div,dl,dt,dd,ul,ol,li,h1,h2,h3,h4,h5,form,fieldset,p,button{margin:0;padding:0}
+  body,h1,h2,h3,h4,input,button,select,option{font-family: 'NotoSansCJKKR','Noto Sans KR', sans-serif; font-size:14px; color:#222;}
+  body{background-color:#fff;*word-break:break-all;-ms-word-break:break-all}
+  img,fieldset,iframe{border:0 none}
+  li{list-style:none}
+  input,select,button{appearance:none; -webkit-appearance:none; vertical-align:bottom; outline:none;}
+  img{vertical-align:top}
+  i,em,address{font-style:normal}
+  label,button{cursor:pointer}
+  button{border:0;margin:0;padding:0;outline:0;}
+  a{color:#222;text-decoration:none; cursor: pointer;}
+  a:hover{color: #222; text-decoration:none}
+  button *{position:relative}
+  legend{*width:0}
+  input::-ms-clear{display:none}
+  input[type="text"]{border: solid 1px #222;height: 48px; padding: 14px 16px; width:100%; font-size: 14px; color: #222;}
+  input[type="radio"], input[type="checkbox"] {display: none;opacity: 0;}
+  ::-webkit-input-placeholder{color:#999; font-size:14px;}
+  :-ms-input-placeholder{color:#999; font-size:14px;}
+  ::placeholder{color:#999; font-size:14px;}
```

Comment:
이 부분을 [styled-reset](https://www.npmjs.com/package/styled-reset) 을 사용해서 간략하게 만들수도 있습니다 :) 물론 하신 방법도 좋습니다!! 패키지 의존성을 줄일 수 있겠네요~


```suggestion
  import reset from "styled-reset";
  
  // ... 
  
  ${reset};
```

Reply:
이거 보면 생각났는데 저번에 종호형이 말씀해주신 [themeprovider](https://styled-components.com/docs/advanced) 이야기 안했네요

Reply:
회의록에 메모메모

Reply:
@crazy-oung 혹시 styled-reset 어떻게 생각하시나요 저는 둘다 괜찮다고는 생각하는데

Reply:
제가 쓰려고 했다가 install 해야해서 의논하고 나서 적용하려고 했거든용 괜찮으시면 styled-reset 쓸까용? 저거는 그냥 항목 몇개 없이 input이랑 기본 설정들만 초기화하려고 간단하게 뺄거 다빼고 넣은거에요